### PR TITLE
Add recipe for helm-rage

### PR DESCRIPTION
**Pull request by mriehl**
### Brief summary of what the package does

A Helm source for raging. Allows you to spice up your commit message with rage comics or various memes.

### Direct link to the package repository

https://github.com/bomgar/helm-rage

### Your association with the package

I'm a contributor.

### Relevant communications with the upstream package maintainer

**None needed**

### Checklist

- [x] I've read [CONTRIBUTING.md](https://github.com/melpa/melpa/blob/master/CONTRIBUTING.md)
- [x] I've built and installed the package using the instructions in the [README](https://github.com/melpa/melpa/blob/master/README.md)

### CODE REVIEW

**Comment by syohex** [comment]
- https://github.com/bomgar/helm-rage/blob/1856f54c6f905fd78dd47a1d91f9bc504b20ed65/helm-rage.el#L26

Use `s.el` function but it is not in package dependencies.

- https://github.com/bomgar/helm-rage/blob/1856f54c6f905fd78dd47a1d91f9bc504b20ed65/helm-rage.el#L30

It looks wrong regexp. `"\.meme"` -> `"\\.meme"`

Missing `meme/` directory in `:files` attribute recipe. Isn't it necessary for this package ?

> I've built and installed the package using the instructions in the README

Did you check to build and install, really ?

**Comment by mriehl** [comment]
You're right, sorry. I thought that firing up the plugin with `use-package load-path` would be sufficient but it turns out that's not true. I've built the package with `package-build` now and corrected the mistakes you pointed out. Thanks a lot.

**Comment by syohex** [comment]
You need to add dependency to Package-Requires header(https://github.com/bomgar/helm-rage/blob/8c6b7cf54efbdb7c14f4a266a62608ceb770464e/helm-rage.el#L6), not Cask.

File: recipes/helm-rage [hunk 1]
```diff
@@ -0,0 +1,3 @@
+(helm-rage :repo "bomgar/helm-rage"
+              :fetcher github
+              :files ("meme" "helm-rage.el"))
```
Comment (syohex) [inline recipes/helm-rage]: Please use `:defaults` instead of `helm-rage.el`. Please read https://github.com/melpa/melpa#recipe-format

**Comment by mriehl** [comment]
@syohex thank you so much for your patience and keen review. I've fixed all the issues now I think, package still builds.

**Comment by syohex** [comment]
Could you add any license ?

**Comment by mriehl** [comment]
@syohex done, we settled for GPLv3. Am I suppose to add the license text to the `:files` metadata or is it acceptable that users will have to browse to the repository to check the license?
I'm also adding a note on the readme, this should be rendered on the melpa package page then.

**Comment by syohex** [comment]
Merged

**Pull request merged**